CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+## How to contribute to Open MPI
+
+General information about contributing to the Open MPI project can be found at the [Contributing to Open MPI webpage](https://www.open-mpi.org/community/contribute/).
+The instructions below are specifically for opening issues and pull requests against Open MPI.
+
+#### **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/open-mpi/ompi/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/open-mpi/ompi/issues/new).
+
+* For more detailed information on submitting a bug report and creating an issue, visit our [Bug Tracking webpage](https://www.open-mpi.org/community/help/bugs.php).
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number in the description.
```

Comment:
Replace the 2nd sentence with: If there's an existing GitHub issue open describing this bug, please include it in the description so that we can close it.